### DISCUSSION TITLE
What are GitHub usage rules regarding multiple personal accounts?

### DISCUSSION THREAD

**Discussion by Potherca** [community/community#57924] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I seem to recal something along the lines of GitHub only allowing a natural person to have _one_ personal account.

But I can no longer find anything regarding usage rules in the GitHub documentation or support apges.

**Can anyone direct me to the correct page where it states whether it is allowed to have more than one personal account?**

**Comment by airtower-luna** [comment, marked as answer]
See here: https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements

This is probably the key part:

> One person or legal entity may maintain no more than one free Account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine).

**Reply by Potherca** [reply]
Thanks! That's exactly what I was looking for!

**Reply by roke-julian-lockhart** [reply]
@airtower-luna, specifically, [`github.com/github/docs/blob/6514e10085aab2cab99ef3e13b886ed61941dbb8/content/site-policy/github-terms/github-terms-of-service.md?plain=1#L79`](https://github.com/github/docs/blob/6514e10085aab2cab99ef3e13b886ed61941dbb8/content/site-policy/github-terms/github-terms-of-service.md?plain=1#L79C3-L79C72:~:text=One%20person%20or%20legal%20entity%20may%20maintain%20no%20more%20than%20one%20free%20Account). [^1]

[^1]: [`reddit.com/r/github/comments/14w872v/comment/o2i83db`](https://www.reddit.com/r/github/comments/14w872v/comment/o2i83db/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)
